### PR TITLE
added individual article page with css, responds well to narrow windo…

### PR DESCRIPTION
**Pull request by J-greaves**
…w or mobile view

### CODE REVIEW

**Comment by J-greaves** [comment]
Apologies, left a couple of console logs in there again!

File: src/api.js [hunk 1]
```diff
@@ -15,3 +15,10 @@ export const getUsers = () => {
     return data;
   });
 };
+
+export const getArticleById = (articleId) => {
+  return ncNews.get(`/articles/${articleId}`).then(({ data }) => {
+    console.log(data);
```
Comment (nc-ali-abdallah) [inline src/api.js:21]: remember to remove console logs.

File: src/App.jsx [hunk 2]
```diff
@@ -11,6 +12,7 @@ function App() {
       <Header />
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/:articleId" element={<ArticlePage />} />
```
Comment (nc-ali-abdallah) [inline src/App.jsx:15]: I'd add something like /articles or /article before this just for better readability in the URL.

**Review by nc-ali-abdallah — COMMENTED** [review]
Awesome stuff, great job on this PR.

Just see the comments I left.

**Pull request merged**